runs/pubmqtt.sh

```diff
@@ -57,6 +57,7 @@ mqttvar["evu/WhExported"]=einspeisungkwh
 mqttvar["evu/WhImported"]=bezugkwh
 mqttvar["housebattery/WhExported"]=speicherekwh
 mqttvar["housebattery/WhImported"]=speicherikwh
+mqttvar["lp/1/SerialNumber"]=lp1Serial
```

Comment:
Should be "MeterSerialNumber", could be misleading without that info.

Reply:
Fully agreed.
Resolved with commit 04cda3305ac8cd7ae8fc8ed48b11c469af618aba